lib/signaling/v2/peerconnection.js

```diff
@@ -753,7 +751,16 @@ PeerConnectionV2.prototype.removeDataTrackSender = function removeDataTrackSende
  */
 PeerConnectionV2.prototype.removeMediaStream = function removeMediaStream(mediaStream) {
   if (this._peerConnection.signalingState !== 'closed') {
-    this._peerConnection.removeStream(mediaStream);
+    var senders = new Map(this._peerConnection.getSenders().filter(function(sender) {
+      return sender.track;
+    }).map(function(sender) {
+      return [sender.track.id, sender];
```

Comment:
I think we should use the MediaStreamTrack itself as the key, not the ID.

Reply:
Also, technically, because of APIs like `replaceTrack`, I think the most appropriate data structure might be `Map<MediaStreamTrack, Set<RTCRtpSender>>`. WDYT?

Reply:
@markandrus Please refer to this comment: https://github.com/twilio/twilio-webrtc.js/pull/19#discussion_r151266957